steamed_hams.sh

```diff
@@ -25,15 +25,13 @@ if [ "$2" = "release" ]; then
     start_datawrite="sleep 0.001s;python3 -O data_write/data_write.py --file-type=hdf5 --enable-raw-acfs --enable-bfiq --enable-antenna-iq; bash;"
     start_usrp_driver="sleep 0.001s; source mode "$2"; usrp_driver > usrp_output.txt; bash"
     start_dsp="sleep 0.001s; source mode "$2"; signal_processing; bash;"
-    start_tids="sleep 0.001s; python3 -O usrp_drivers/set_affinity.py; bash;"
```

Comment:
Does affinity not need to happen anymore?

Reply:
no, it wasnt working well for tx and it wasnt worth the effort at the time.

Reply:
no. i didnt work well for tx issues.